src/metrics/include/metrics/operation.hpp

```diff
@@ -255,10 +255,6 @@ class OperationImpl final : private boost::noncopyable {
     }
 
 private:
-    /*
-     * Actor count and phase number will be used in Poplar metrics. Right now they
-     * are unused.
-     */
```

Comment:
I removed this comment because [the commit that introduce this comment doesn't actually add actor count](https://github.com/mongodb/genny/commit/c66113eaa8652f2375accbc24429a39da0910a75#diff-9951fc5c4b608ec55c6d5473fa06f5802e6b731eca2292a233fd8d46c3dd4365R224-R235), and phase number has been deleted in [a previous commit](https://github.com/mongodb/genny/commit/9e0031086b5227f41660a1344465ccb1a46bec1c#diff-9951fc5c4b608ec55c6d5473fa06f5802e6b731eca2292a233fd8d46c3dd4365L268).